pyproject.toml

```diff
@@ -0,0 +1,30 @@
+[tool.poetry]
+name = "nums"
+version = "0.2.3"
+description = "A numerical computing library for Python that scales."
+authors = ["NumS Development Team <elibol@cs.berkeley.edu>"]
+license = "Apache-2.0"
+readme = "README.md"
+repository = "https://github.com/nums-project/nums"
+documentation = "https://nums.readthedocs.io/en/latest/"
+
+[tool.poetry.dependencies]
```

Comment:
For dependencies, could we specify them all in one place? With this portion of the code, it appears we need to specify dependencies in multiple places.